<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>基于Arduino的盆栽培育系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget:pane{
border: 1px solid rgba(173, 173, 173, 255); 
top: -1px;
}

QTabBar::tab{
height:30px; 
min-width:100px;
}

QTabBar::tab:selected{
height:35px;
border:1px solid rgba(173, 173, 173, 255); 
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(173, 173, 173, 255), stop:1 rgba(255, 255, 255, 255)); border-bottom-color: none;
}

QTabBar::tab:!selected{
height:30px;
margin-top: 5px;
border:1px solid rgba(173, 173, 173, 255); 
border-bottom: 1px solid rgba(173, 173, 173, 255);
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>系统信息</string>
       </attribute>
       <widget class="QLabel" name="temp_img">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>50</y>
          <width>64</width>
          <height>64</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/temp.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="line_top">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>36</y>
          <width>581</width>
          <height>1</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="baseInfoTittle">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>4</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>基础信息</string>
        </property>
       </widget>
       <widget class="QLabel" name="line_bootom">
        <property name="geometry">
         <rect>
          <x>7</x>
          <y>252</y>
          <width>581</width>
          <height>1</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>功能控制</string>
        </property>
       </widget>
       <widget class="QLabel" name="fun_tittle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>功能控制</string>
        </property>
       </widget>
       <widget class="QLabel" name="humidity_img">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>50</y>
          <width>64</width>
          <height>64</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/humidity.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="light_img">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>64</width>
          <height>64</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/light.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="pump_img">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>160</y>
          <width>64</width>
          <height>64</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/pump.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="temp_text">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>120</y>
          <width>60</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>0℃</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="humidity_text">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>120</y>
          <width>60</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>0%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="lightBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>打开光照</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pumpBtn">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>270</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>浇水</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pump_time">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>270</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="pump_unit">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>270</y>
          <width>31</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string> 秒</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="plant">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>128</width>
          <height>128</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/plant.png</pixmap>
        </property>
       </widget>
       <widget class="QLineEdit" name="pump_threshold">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>320</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>50</string>
        </property>
       </widget>
       <widget class="QLabel" name="pump_unit_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>320</y>
          <width>71</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>自动浇水</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="pump_unit_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>320</y>
          <width>81</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>阈值低于</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QToolButton" name="autowaterBtn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>320</y>
          <width>161</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>自动浇水关闭</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/switchOff.png</normaloff>:/images/switchOff.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>串口配置</string>
       </attribute>
       <widget class="QComboBox" name="ports">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>串口号:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>波特率:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="bauds">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
       <widget class="QToolButton" name="serialBtn">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
